app/connector/support/util/src/main/java/io/syndesis/connector/support/util/KeyStoreHelper.java

```diff
@@ -32,6 +40,11 @@
  */
 public class KeyStoreHelper {
 
+    /**
+     * The password for the java runtime's default keystore
+     */
+    public static final String DEFAULT_KEYSTORE_PASSWD = "changeit";
+
     private static final SecureRandom SECURE_RANDOM = new SecureRandom();
```

Comment:
This optimization is not needed, usages of this field should be replaced with `java.util.concurrent.ThreadLocalRandom`